tailwind.config.js

```diff
@@ -9,6 +9,8 @@ export default {
       colors: {
         blue: {
           100: '#458FF6',
+          200: '#5A5F70',
+          300: '#1C202E'
```

Comment:
We should define new colors instead of overriding the existing ones (tailwind already comes with a `blue-300` color).